package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@openactive/dataset-utils",
+  "version": "1.0.0",
+  "description": "Crawls OpenActive data-catalogs and returns an array of dataset sites",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openactive/dataset-utils.git"
+  },
+  "keywords": [
+    "dataset-utils",
+    "openactive"
+  ],
+  "author": "Civ Sivakumaran",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/openactive/dataset-utils/issues"
+  },
+  "homepage": "https://github.com/openactive/dataset-utils#readme",
+  "dependencies": {
+    "axios": "^1.4.0"
```

Comment:
Need imports for these:

```
const { Handler } = require('htmlmetaparser');
const { Parser } = require('htmlparser2');
```

Reply:
done

Reply:
(Can't click resolved for some reason) **RESOLVED**